iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergTableOperations.java

```diff
@@ -114,6 +116,12 @@ public Response listTable(
   @Produces(MediaType.APPLICATION_JSON)
   @Timed(name = "create-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "create-table", absolute = true)
+  @AuthorizationExpression(
```

Comment:
Could you keep the `AuthorizationExpression` the same with the ones in Gravitino table operations?